go-controller/pkg/ovn/libovsdbops/model.go

```diff
@@ -120,6 +120,7 @@ func copyIndexes(model model.Model) model.Model {
 	case *nbdb.LoadBalancerGroup:
 		return &nbdb.LoadBalancerGroup{
 			UUID: t.UUID,
+			Name: t.Name,
```

Comment:
@dceara Since LB_Group is indexed by name, we need to add that here

Reply:
Super NIT: But could you please put this little bit int another commit for readability?

Reply:
ack. good point. will do.

Reply:
done.